apps/backend/src/app/api/latest/internal/failed-emails-digest/route.ts

```diff
@@ -50,28 +50,34 @@ export const POST = createSmartRouteHandler({
 
     let anyDigestsFailedToSend = false;
     for (const failedEmailsBatch of failedEmailsByTenancy.values()) {
+      if (!failedEmailsBatch.tenantOwnerEmails.length) {
+        continue;
+      }
+
       const viewInStackAuth = `<a href="${dashboardUrl}/projects/${encodeURIComponent(failedEmailsBatch.projectId)}/emails">View all email logs on the Dashboard</a>`;
       const emailHtml = `
         <p>Thank you for using Stack Auth!</p>
         <p>We detected that, on your project, there have been ${failedEmailsBatch.emails.length} emails that failed to deliver in the last 24 hours. Please check your email server configuration.</p>
         <p>${viewInStackAuth}</p>
         <p>Last failing emails:</p>
         ${failedEmailsBatch.emails.slice(-10).map((failedEmail) => {
-          const escapedSubject = escapeHtml(failedEmail.subject).replace(/\s+/g, ' ').slice(0, 50);
-          const escapedTo = failedEmail.to.map(to => escapeHtml(to)).join(", ");
-          return `<div><p>Subject: ${escapedSubject}<br />To: ${escapedTo}</p></div>`;
-        }).join("")}
+        const escapedSubject = escapeHtml(failedEmail.subject).replace(/\s+/g, ' ').slice(0, 50);
+        const escapedTo = failedEmail.to.map(to => escapeHtml(to)).join(", ");
+        return `<div><p>Subject: ${escapedSubject}<br />To: ${escapedTo}</p></div>`;
+      }).join("")}
         ${failedEmailsBatch.emails.length > 10 ? `<div>...</div>` : ""}
       `;
       if (query.dry_run !== "true") {
         try {
-          await sendEmail({
-            tenancyId: internalTenancy.id,
-            emailConfig,
-            to: failedEmailsBatch.tenantOwnerEmail,
-            subject: "Failed emails digest",
-            html: emailHtml,
-          });
+          for (const tenantOwnerEmail of failedEmailsBatch.tenantOwnerEmails) {
+            await sendEmail({
+              tenancyId: internalTenancy.id,
+              emailConfig,
+              to: tenantOwnerEmail,
+              subject: "Failed emails digest",
+              html: emailHtml,
+            });
+          }
```

Comment:
`to` actually takes a string array, so you don't have to call the function multiple times